src/components/UserMenu.js

```diff
@@ -35,7 +42,7 @@ const UserMenu = () => {
 							</span>
 						</div>
 						{/* Name */}
-						<div className="text-sm pl-2">
+						<div className="text-sm pl-2 hidden md:block">
 							{user.name.split(' ')[0]}
```

Comment:
No error handling, this could lead to runtime crash if `user` has no last name i guess ! or worse, user has no name at all